snapshotter/cmd/naive/Makefile

```diff
@@ -13,7 +13,7 @@
 
 all: naive_snapshotter
 
-naive_snapshotter: *.go
+naive_snapshotter: *.go ../../../go.mod ../../../go.sum
```

Comment:
I would rather use this instead of `../../../`
```
GOMOD := $(shell go env GOMOD)
GOSUM := $(GOMOD:.mod=.sum)
```

Reply:
I had no idea that existed; that's great!  Will definitely switch.

Reply:
Done!